packages/block-editor/src/components/block-navigation/expander.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * WordPress dependencies
+ */
+import { chevronRightSmall, Icon } from '@wordpress/icons';
+export default function BlockNavigationExpander( { onClick } ) {
+	return (
+		// Keyboard events are handled by TreeGrid see: components/src/tree-grid/index.js
+		// eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions
+		<span
```

Comment:
The implementation as a pseudo-element in the w3 example was really surprising to me. 

I guess that makes it completely hidden from the accessibility tree, in which case the span could have `aria-hidden="true"` specified to replicate that.

It might be worth adding some comments to the component explaining why this is implemented like it is—I imagine many devs with good intentions might be tempted to change it to a `Button`.

Reply:
Good call, I've updated this in https://github.com/WordPress/gutenberg/pull/32117/commits/f4b5a5f3527628d8969938ab3024189a0ef37348

For folks 👀  we can see the pseudo element in https://www.w3.org/TR/wai-aria-practices/examples/treegrid/treegrid-1.html
 
<img width="1446" alt="Screen Shot 2021-06-14 at 9 25 57 AM" src="https://user-images.githubusercontent.com/1270189/121926692-14062b80-ccf3-11eb-9c68-84fd12985ffb.png">